Modify sections on Vulnerability Management: security patches and antivirus (#240)
sa-tre/satre-specification: docs/source/pillars/data_management.md

  - Statement
  - Guidance
  - Importance

* - 3.5.1.
  - You must have a process in place for applying security updates to all software that forms part of the TRE infrastructure.
  - This includes any software used for remote desktop portals, databases, webapps, creating and destroying compute infrastructure, configuration management, or software used for monitoring the TRE.
  - Mandatory
* - 3.5.2.

>   - You must also run regular anti-virus/malware scans on all TRE systems where infection could be a problem.

Reviewer: ```suggestion
  - You must run regular anti-virus/malware scans on all TRE systems where infection could be a problem.
```

  - Guidance
  - Importance
* - 3.5.1.

>   - You should keep all TRE computing infrastructure up-to-date with security patches and antivirus (if appropriate).

Reviewer 2: ```suggestion
  - You should keep all TRE computing infrastructure up to date with security patches and antivirus (if appropriate).
```

Not being used as an adjective here.

>   - You should keep all TRE computing infrastructure up-to-date with security patches and antivirus (if appropriate).

Reviewer 2: Are we saying antivirus should be kept up to date, or that we should also use that?

Reviewer 3: Rephrased.

>   - This might involve scheduling regular automated scanning and application of updates.

Reviewer 2: ```suggestion
  - This might involve scheduling regular automated scanning and updating.
```

Application here is a little confusing.
